packages/mobile/src/components/core/TextInput.tsx

```diff
@@ -89,9 +90,6 @@ const useStyles = makeStyles(({ typography, palette, spacing }) => ({
     width: spacing(4),
     height: spacing(4)
   },
-  placeholderText: {
-    color: palette.neutralLight7
```

Comment:
is subdued text color not in palette?

Reply:
probably is but i wanted to start using harmony